terraform/projects/cloudbank.tfvars

```diff
@@ -0,0 +1,25 @@
+prefix     = "cb"
+project_id = "cb-1003-1696"
+
+core_node_machine_type = "n1-highmem-4"
+
+enable_network_policy    = true
+config_connector_enabled = false
```

Comment:
I noticed you have the `config_connector_enabled = false` here and in the meom-ige cluster but it is `true` is the pilot-hubs.tfvars, can you expand about that difference?

Reply:
I've left comments here, and also added a new piece of documentation about the design of the GKE clusters